pkg/server/commands/list_stores.go

```diff
@@ -52,6 +52,7 @@ func (q *ListStoresQuery) Execute(ctx context.Context, req *openfgav1.ListStores
 
 	opts := storage.ListStoresOptions{
 		IDs:        storeIDs,
+		Name:       req.GetName(),
```

Comment:
@kalleep can we temporarily point `github.com/openfga/api/proto` in this PR to https://github.com/openfga/api/pull/211  so that tests can actually run?

Reply:
Done